src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts

```diff
@@ -669,39 +669,45 @@ export const walkthroughs: GettingStartedWalkthroughContent = [
 	},
 	{
 		id: 'NewWelcomeExperience',
-		title: localize('gettingStarted.setup.title', "Get Started with VS Code"),
-		description: localize('gettingStarted.setup.description', "Customize your editor, learn the basics, and start coding"),
+		title: localize('gettingStarted.new.title', "Get started with VS Code"),
+		description: localize('gettingStarted.new.description', "Supercharge coding with AI"),
 		isFeatured: false,
 		icon: setupIcon,
 		when: '!isWeb',
-		walkthroughPageTitle: localize('gettingStarted.setup.walkthroughPageTitle', 'Setup VS Code'),
+		walkthroughPageTitle: localize('gettingStarted.new.walkthroughPageTitle', 'Set up VS Code'),
 		content: {
 			type: 'steps',
 			steps: [
 				{
 					id: 'copilotSetup.chat',
-					title: localize('gettingStarted.agentMode.title', "Agent Mode"),
-					description: localize('gettingStarted.agentMode.description', "Tackle complex, multi-step tasks with AI"),
+					title: localize('gettingStarted.agentMode.title', "Agent mode"),
+					description: localize('gettingStarted.agentMode.description', "From idea to implementation, agent mode reasons through the problem, plans next steps, and applies changes"),
 					media: {
 						type: 'svg', altText: 'VS Code Copilot multi file edits', path: 'multi-file-edits.svg'
 					},
 				},
 				{
 					id: 'copilotSetup.inline',
 					title: localize('gettingStarted.nes.title', "Next Edit Suggestions"),
-					description: localize('gettingStarted.nes.description', "Your next move, predicted while you code"),
+					description: localize('gettingStarted.nes.description', "Get code suggestions, predicting your next edit while you're in the flow of coding."),
 					media: {
 						type: 'svg', altText: 'Next Edit Suggestions', path: 'ai-powered-suggestions.svg'
 					},
 				},
 				{
 					id: 'copilotSetup.customize',
-					title: localize('gettingStarted.customize.title', "Customize"),
-					description: localize('gettingStarted.customize.description', "Choose your model, tools, and personalized instructions\n{0}", Button(localize('signUp', "Set up AI in VS Code"), 'command:workbench.action.chat.triggerSetup')),
+					title: localize('gettingStarted.customize.title', "Personalize"),
+					description: localize('gettingStarted.customize.description', "Swap models, add agent mode tools, and create instructions to tailor the AI to how you work.\n{0}", Button(localize('signUp', "Set up AI"), 'command:workbench.action.chat.triggerSetup')),
 					media: {
-						type: 'svg', altText: 'Customize', path: 'multi-file-edits.svg'
+						type: 'svg', altText: 'Personalize', path: 'multi-file-edits.svg'
 					},
 				},
+				{
+					id: 'newCommandPaletteTask',
+					title: localize('newgettingStarted.commandPalette.title', "All VS Code commands within reach"),
```

Comment:
IIRC this title spilled over to a second line as well so suggest cutting "VS Code"
```suggestion
					title: localize('newgettingStarted.commandPalette.title', "All commands within reach"),
```